configure.ac

```diff
@@ -841,6 +841,11 @@ then
     AM_CFLAGS="$AM_CFLAGS -DNO_AES_CBC"
 fi
 
+if test "$ENABLED_AESCBC" != "no"
+then
+    AM_CFLAGS="$AM_CFLAGS -DHAVE_AES_CBC"
```

Comment:
In ./wolfssl/wolfcrypt/settings.h, HAVE_AES_CBC is defined if NO_AES_CBC is not defined:

```
#ifndef NO_AES_CBC
    #undef  HAVE_AES_CBC
    #define HAVE_AES_CBC
#endif
```

Are you sure that configure needs to define it again?

Reply:
I missed that.  I removed that from configure.ac.  Thanks!